.github/workflows/build.yml

```diff
@@ -140,9 +140,16 @@ jobs:
               `;
             }, '' );
             const commentBody = `${ TEST_COMMENT_INDICATOR }
-            Are you an Automattician? You can now test your Pull Request on WordPress.com. On your sandbox, run
+            Are you an Automattician? Please test your changes on all WordPress.com environments to help mitigate accidental explosions.
+
+            - To test on WoA, go to the Plugins menu on a WordPress.com Simple site. Click on the "Upload" button and follow the upgrade flow to be able to upload, install, and activate [the Jetpack plugin](https://jetpack.com/download-jetpack-beta/). Once the plugin is active, go to Jetpack > Jetpack Beta > Jetpack and enable the \`${ BRANCH_NAME }\` branch.
```

Comment:
This strikes me as a bit long. Personally I'd go back to what you had earlier, and include a doc ref for even more verbose details in case someone doesn't know how to add a plugin yet.

Reply:
It is a bit long indeed. It may, however, stand out a bit more like this. In the spirit of compromising, I think it is a good middleground based on [the conversation](https://github.com/Automattic/jetpack/pull/31266#discussion_r1223501147) we had with @danielbachhuber above.

We can iterate in follow-up PRs based on the feedback we'll get from everyone once this is merged.

Reply:
Seems like it'll be TL;DR to me. How many people do we really expect to get this message, be able to follow it, but not know how to install a plugin?

Reply:
I think that can be the case for folks used to Jetpack or WordPress.com Simple development (or getting started with those ), but not familiar with the Atomic infrastructure.